.github/workflows/pull_request.yml

```diff
@@ -51,13 +52,57 @@ jobs:
       - name: Upload Coverage Report
         uses: codecov/codecov-action@v1
 
+  sanity-test:
+    # to ignore builds on release
+    if: ${{ (github.event.ref_type != 'tag') }}
+    runs-on: ubuntu-latest
+    needs: ['lint', 'unit-test']
```

Comment:
@niladrih Running sanity test parallel is causing test to get timeout(May be due to less resources)

Reply:
One of the tests failed a due to 404 error.

2 other sanity-tests running in #272 and #273 passed.

Likely cause is test flakiness.